docs/paper.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'scores: A Python package for verifying accuracy using xarray and pandas'
+tags:
+  - Python
+  - geoscience
+  - verification
+  - science
+  - earth system science
+  - statistics
+authors:
+  - name: Tennessee Leeuwenburg
+    orcid: 0009-0008-2024-1967
+    affiliation: 1 
+    corresponding: true # (This is how to denote the corresponding author)    
+  - name: Nicholas Loveday
+    affiliation: 1
+affiliations:
+ - name: Bureau of Meteorology, Australia
+   index: 1
+date: 6 September 2023
+bibliography: paper.bib 
+
+---
+
+# Summary
+
+`scores` is a package containing mathematical functions for the verification, evaluation, and optimisation of model outputs and predictions. It primarily supports the geoscience and earth system science communities. `scores` is focused on supporting xarray datatypes for earth system data. It has wide potential application in machine learning, and domains other than meteorology, geoscience and weather. It also aims to be compatible with pandas, geopandas, pangeo and work with NetCDF4, Zarr, hdf5 and GRIB data sources among others. 
+
+All of the scores and metrics in this package have undergone a thorough statistical and scientific review. Every score has a companion Jupyter Notebook demonstrating its use in practise.
+
+At the time of writing, the scores contained in this package are: MSE, MAE, RMSE, FIRM [@Taggart:2022], CRPS, the FlipFlop index [@Griffiths:2019] and the Murphy score [@Ehm:2016]. It also includes the Diebold-Mariano statistical test [@Hering:2011] and [@Harvey:1997]. 
```

Comment:
This is a matter of priority and time, I'm not sure whether it will make the cutoff before publication. I'll exclude this for now but it would of course be nice if we could include it.